form.yml.erb

```diff
@@ -25,7 +18,6 @@ attributes:
     min: 0
```

Comment:
Let's get rid the practice of having 0 minimums. It doesn't make sense. Blank is valid a use case sometimes, but not zero.

Reply:
Good call. Missed it initially.
Ready for review.